domains/cutedog5695.json

```diff
@@ -6,6 +6,11 @@
         "twitter": "cutedog5695"
     },
     "record": {
-        "CNAME": "cutedog5695.github.io"
-    }
+        "CNAME": "cutedog5695.github.io",
```

Comment:
CNAME cannot be used in conjunction with other records and vice versa.

Reply:
so basically i cant use CNAME if i want to add other records?